### PR TITLE
[NO-JIRA] Adding textAlignment and lineBreakMode to BPKTappableLinkLable

### PR DESCRIPTION
**Pull request by adborbas**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [ ] `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -24,15 +29,16 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 **Breaking:**
 
 - Backpack/Icon:
```
Comment (k0nserv) [inline UNRELEASED.md]: Can you revert the changes below this line, they look incorrect.

Comment (adborbas) [reply UNRELEASED.md]: Sure
